airmozilla/new/templates/new/_email_new_event.html

```diff
@@ -23,6 +23,19 @@
   Video privacy: <b>{{ event.get_privacy_display() }}</b>
 </p>
 
+<p> 
+  Channels:
+  {% if event.channels.all().count() > 0 %} 
+    <ul>
+    {% for channel in event.channels.all() %}
+      <li>{{ channel | safe}}</li>
+    {% endfor %}
+    </ul>
+  {% else %} 
+     <p> No channels </p>
+  {% endif %}
+</p>
+
```

Comment:
Why wrap everything in `<p></p>`? The possible outcomes are:
```html
<p>
  Channels:
    <p> No channels </p>
</p>
```
or 
```html
<p>
  Channels:
    <ul>
      <li>SomeChannel</li>
    </ul>
</p>
```
That's not how `<p>` tags are supposed to be used.